WalletWasabi.Fluent/Controls/TagsBox.axaml.cs

```diff
@@ -272,8 +272,6 @@ private void CheckIsCurrentTextValid()
 			IsCurrentTextValid = !string.IsNullOrEmpty(correctedInput);
 			return;
 		}
-
-		throw new InvalidOperationException($"Invalid configuration! {nameof(Suggestions)} are not set!");
```

Comment:
If the `RestrictInputToSuggestions` is set to `true` meaning only those items could be accepted that exist in the `Suggestions`. But if the `Suggestions` are not set, then it is an incorrect usage of the control, not possible to accept items from the suggestions if they don't exist.

That is why the exception was there.

The goal would be that this function does not get called when the dialog is closing. So all the subscriptions that happen in the `OnApplyTemplate` method should be canceled when they are no longer needed. This can be achieved by calling `_compositeDisposable.Dispose();` at a specific point. The only question is when is that specific point. `OnDetachedFromVisualTree` would be a could point. But refactoring is required for the control to be usable when it gets re-attached to the visual tree.

Reply:
@Onyx2406 please add `IsCurrentTextValid = false` so that the method will fall through (as seen in #7169)